react/ProductSummaryContext.tsx

```diff
@@ -122,16 +122,16 @@ export function reducer(state: State, action: Action): State {
   }
 }
 
-const buildProductQuery = ((product: Product) => {
+const buildProductQuery = ((product: Product, listName?: string) => {
```

Comment:
I'm not sure, but if there is something new to add in the future to this function will need to add a 3rd parameter, what do you think about using an object as a second parameter?